### PR TITLE
Add GLTO-ERC20 bridged via INJ

### PR DESCRIPTION
**Pull request by Rizbe**
### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - The `description` and/or `extended_description` of the asset in the Chain Registry is informative.
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)
   - Note that it is recommended to include an X (fka, Twitter) profile URL with each asset.

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [ ] Ready -- A liquidity pool has been created. The pool ID is: ______
- [x] Soon -- A pool will be created. (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).)
  - [ ] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Supercharged Liquidity pools cannot be created via Osmosis Zone app)
- [ ] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### CODE REVIEW

**Comment by Rizbe** [comment]
@JeremyParish69 after this is merged do I need to do anything here: https://github.com/osmosis-labs/osmosis-frontend/tree/stage

Or is this the last step before it's added to Osmo FE?

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -2516,6 +2516,13 @@
...
       "path": "transfer/channel-326/stutia",
       "osmosis_verified": true
+    },
+    {
+      "chain_name": "injective",
+      "base_denom": "peggy0xd73175f9eb15eee81745d367ae59309Ca2ceb5e2",
+      "path": "transfer/channel-122/peggy0xd73175f9eb15eee81745d367ae59309Ca2ceb5e2",
+      "twitter_URL": "https://twitter.com/Gelotto2",
+      "osmosis_verified": false
     }
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2526]: Please add "osmosis_unlisted": true

**Comment by JeremyParish69** [comment]
> @JeremyParish69 after this is merged do I need to do anything here: https://github.com/osmosis-labs/osmosis-frontend/tree/stage
> 
> Or is this the last step before it's added to Osmo FE?

No, frontend code will not need any updating. Once validated and a liquidity exists, it will appear on frontend after updating this repo again (to remove "osmosis_unlisted").

File: osmosis-1/osmosis.zone_assets.json (hunk 1 above)
Comment (Rizbe) [reply osmosis-1/osmosis.zone_assets.json:2527]: Done

**Comment by Rizbe** [comment]
> > @JeremyParish69 after this is merged do I need to do anything here: https://github.com/osmosis-labs/osmosis-frontend/tree/stage
> > Or is this the last step before it's added to Osmo FE?
> 
> No, frontend code will not need any updating. Once validated and a liquidity exists, it will appear on frontend after updating this repo again (to remove "osmosis_unlisted").

Transmuter pool made: pool ID: 1445

**Comment by JeremyParish69** [comment]
Are we going to need an external interface for transferring the asset back and forth between Injective and Osmosis? I believe all other Injective assets have required this

**Comment by Rizbe** [comment]
INJ will not be adding GLTO to their bridge page to enable transfer from their UI. I was able to manually IBC to Osmo, can we have it IBC like other assets in the asset page?

**Comment by JeremyParish69** [comment]
My guess is that it won't work, but we can validate using the preview

**Comment by Rizbe** [comment]
@JeremyParish69 Looks like you were right, I am not able to IBC:
```
Broadcasting transaction failed with code 8 (codespace: ). Log: pubKey does not match signer address
```

Prob due to INJ using a different key path vs most of Cosmos

**Comment by JeremyParish69** [comment]
> @JeremyParish69 Looks like you were right, I am not able to IBC:
> 
> ```
> Broadcasting transaction failed with code 8 (codespace: ). Log: pubKey does not match signer address
> ```
> 
> Prob due to INJ using a different key path vs most of Cosmos

Ya, so until Osmosis has this built, we'll need an alternative transfer UI. I think some others from Injective has used TFM.

**Pull request merged**